tests/Testdata/pos2.json

```
@@ -0,0 +1,11 @@
+{
+	"fullWidth": null,
```

Please indent the lines evenly.

```suggestion
  "fullWidth": null,
```